src/config.ts

```diff
@@ -13,6 +13,7 @@ export const feedbackUrl = env.NEXT_PUBLIC_FEEDBACK_URL;
 export const kgInferenceBaseUrl = env.NEXT_PUBLIC_KG_INFERENCE_BASE_URL;
 export const thumbnailGenerationBaseUrl = env.NEXT_PUBLIC_THUMBNAIL_GENERATION_BASE_URL;
 export const synthesisUrl = env.NEXT_PUBLIC_SYNTHESIS_URL;
+export const entityCoreUrl = env.NEXT_PUBLIC_ENTITY_CORE;
```

Comment:
Probably it's worth adding a `_URL` prefix to clearly state what's the type of expected value.